app/destination/page.js

```diff
@@ -1,40 +1,59 @@
 "use client";
 
-import { useState } from 'react';
+import { useState } from "react";
+import PlanetCard from "./PlanetCard";
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
+import PlanetWishlistItem from "./PlanetWishlistItem/PlanetWishlistItem";
+
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
 
 // TASK - React 1 week 2
 // Move this to its own file
-const PlanetWishlistItem = ({
-  name,
-  onRemove,
-  thumbnail,
-}) => {
-  return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
-
 
 export const Destinations = () => {
   const [selectedPlanets, onAddPlanet] = useState([]);
```

Comment:
You could initialise the state with your array of planets

Reply:
i did it